src/App.js

```diff
@@ -182,6 +182,19 @@ class App extends Component {
       this.store.write('themeId', themeId);
     }
   }
+  
+  handleConfigIdChange = (id) => {
+    this.configId = id;
+    const list = this.store.read('whitelist' + this.configId) || [];
+    const layout = this.store.read('layout' + this.configId) || [];
+    this.setState({
+      whitelist: list,
+      layout: layout
+    });
+    for (let i = 0; i < list.length; i++) {
+      this.requestDeviceStatus(list[i]);
+    }
```

Comment:
I don't understand yet but if I call handleDeviceListChange here, layout is messy with blank parts at the top where the previous widgets were